content/en/real_user_monitoring/mobile_and_tv_monitoring/data_collected/android.md

```diff
@@ -36,6 +36,15 @@ The following diagram illustrates the RUM event hierarchy:
 
 {{< img src="real_user_monitoring/data_collected/event-hierarchy.png" alt="RUM Event hierarchy" style="width:50%;" >}}
 
+## Application launch
+
+During initialization, the RUM Android SDK creates a view called "ApplicationLaunch". This view's start time matches the start of the Android process, and can be used to track your application launch time.
+
+The ApplicationLaunch view includes any logs, actions, and resources created before your first call to `startView`. Use the duration of this view to determine time to first view. This view has an action, `application_start`, with a duration equal to the amount of time from process start until the call to `applicationDidBecomeActive`.
```

Comment:
`applicationDidBecomeActive` is an iOS callback and won't exist on Android, so we should remove it. 

We can state that the `application_start` duration will be equal to the two bullet points below, depending on Android version.

Reply:
Hi @plousada, I've updated it. What do you think?